apps/kevin-malone/bot/common/threads/shared_steps.py

```diff
@@ -227,7 +205,14 @@ async def send(self, message, user_id):
 
     async def save(self, message, guild_id, user_id):
         await self.verify_message(message)
-        await self.save_authenticated_account()
+        if self.update:
+            await self.save_authenticated_account()
+
+    async def handle_emoji(self, raw_reaction):
+        # skips twitter verification
+        if SKIP_EMOJI in raw_reaction.emoji.name and not self.update:
```

Comment:
Doesn't this make the twitter address required? I can't skip the step when testing. We should make this really easy to skip or remove the verification